packages/apps/halo-app/src/vault/iframe.ts

```diff
@@ -210,7 +210,7 @@ export const startIFrameRuntime = async (createWorker: () => SharedWorker): Prom
       await startShell(config, iframeRuntime.shell, shellClientProxy, origin);
     }
 
-    info.forEach((message) => console.log(message, cssLogStyle));
+    info.forEach((message) => log.info(message, cssLogStyle));
```

Comment:
I don't think the style will work the same way here, but either way I don't think this iframe is really used anymore. Perhaps just comment this out for now